CIPs/cip-0024.md

```diff
@@ -20,9 +21,9 @@ Enable the activation of the Celo Churrito Hardfork specification which includes
 Add support for protocol-impacting changes introduced in Geth in a hardfork called `Churrito`.
 
 This document proposes the following blocks at which to implement these changes in the Celo networks:
-- `TBD` on Baklava Testnet ()
-- `TBD` on Alfajores Testnet ()
-- `TBD` on Celo Mainnet ()
+- `2598139` on Baklava Testnet (Tuesday November 24th, 2020)
+- `2320192` on Alfajores Testnet (Tuesday December 1st, 2020)
+- `3976252` on Celo Mainnet (Tuesday December 8th, 2020)
```

Comment:
Could you add a tldr of how you came up with the math for these blocks in the PR description?

Reply:
Here are my notes:
5 second block time
12 Blocks per minute * 60 minutes * 24 hours * days

Baklava
11 Days
2,408,059 + 190080 = 2598139
Tuesday 24th

Alfajores
18 Days
2,009,152 + 311040 = 2320192
December 1st

Mainnet
25 Days
3,544,252 + 432000 = 3976252
December 8th